manifests/0000_26_cloud-controller-manager-operator_11_deployment.yaml

```diff
@@ -103,6 +103,34 @@ spec:
           - mountPath: /etc/kubernetes
             name: host-etc-kube
             readOnly: true
+      - args:
+        - --secure-listen-address=0.0.0.0:9192
+        - --upstream=http://127.0.0.1:9191/
```

Comment:
```suggestion
        - --secure-listen-address=0.0.0.0:9258
        - --upstream=http://127.0.0.1:9257/
```

Reply:
Thanks very much for the review Joel, I've implemented your changes and a volume I was missing.